src/server/commands.rs

```diff
@@ -160,6 +160,13 @@ pub struct CompositionInitializeParams {
 #[serde(rename_all = "camelCase")]
 pub struct ValueFilterParams {
     pub text_document: lsp::TextDocumentIdentifier,
-    pub name: String,
```

Comment:
This is me bringing it closer to the spec.
Which defines what is the `ValueFilterParams` versus the `TagValueFilterParams`